### ISSUE TITLE
several issues:

### ISSUE THREAD

**Issue by Xiaohua-Shen** [kui-shell/AnimalApp#9]
1.  I tried to start proxy with KUBECONFIG, but sees not work
```
git clone https://github.com/kui-shell/plugin-kubeui && cd plugin-kubeui
npm ci
KUBECONFIG=xxxx npm run proxy &  
npm run webpack 
```
The output is like below, seems the KUBECONFIG env variable is not there.
```
❯
ready
Successfully connected to your cloud. For next steps, try this command: getting started
❯
env
KUI=true
TERM=xterm-color
KUI_REPL_MODE=stdout
KUI_EXEC_OPTIONS={"noHistory":true,"rethrowErrors":true,"type":2,"env":{},"nested":true,"isProxied":true,"credentials":{},"rawResponse":true,"language":"en-US"}
__CF_USER_TEXT_ENCODING=0x1F5:0x19:0x34
PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin
LOCALE=en-US
DEVMODE=true
PWD=/
SHLVL=3
HOME=/Users/shenxh
KUI_HEADLESS=true
_=/usr/bin/env
❯
kubectl get pod
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```
and the UI console has many error like 
```
Not allowed to load local resource: file:///Applications/Utilities/Terminal.app/Contents/Resources/Fonts/SFMono-Light.otf
```

2. if I stop the `npm run webpack ` and run again, when access http://localhost:9080, will show errors like below
<img width="1599" alt="屏幕快照 2019-10-25 上午10 09 36" src="https://user-images.githubusercontent.com/19463092/67537847-b7b25a80-f70f-11e9-8209-c427187bae67.png">
clean dist and node-modules and rerun `npm ci` can resolve this issue.

**Comment by Xiaohua-Shen** [comment]
the shell start at root directory `/`, can it be set to user's home directory?

**Comment by Xiaohua-Shen** [comment]
now `npm run` include steps of build and run,  can it be separated?  Since it can be build once and restart running multiple times.

**Comment by Xiaohua-Shen** [comment]
user need to switch kubecontext.  in current mode, user need to restart proxy with new kubecontext env variables, are there any convenient way to do it?

**Comment by Xiaohua-Shen** [comment]
```ContentSecurityPolicy: dev-server default-src 'none'; style-src 'self' 'unsafe-inline'; img-src 'self' file: 'nonce-kuiDefaultNonce' data:; script-src 'self' 'nonce-kuiDefaultNonce' 'strict-dynamic' 'unsafe-eval'; font-src 'self' file:; connect-src 'self' http://localhost:8081 http://localhost:9953 ws://localhost:8081 ws://localhost:9080```
port 9953 is used for what purpose?

**Comment by starpit** [comment]
thanks for the bug reports. the "launch webpack second time" bug seems to be a bug in webpack. for now, i have merged a workaround: https://github.com/kui-shell/plugin-kubeui/pull/39

**Comment by starpit** [comment]
CSP fixed by #41
